File: samples/sols/java-ml/sum.java
import java.io.File;
import java.io.FileWriter;
import java.util.Scanner;

class sum
{
    public static void main (String[] arguments)
    {
        try {
            int[] arr = new int[100000000];
            Scanner scanner = new Scanner(new File("sum.in"));
            FileWriter writer = new FileWriter("sum.out");
            int result = scanner.nextInt() + scanner.nextInt();
            writer.write(String.valueOf(result) + "\r\n");
            writer.close();
        } catch (Exception ex) {}
    }
}

File: samples/sols/python-ml/sum.py
fin = open('sum.in', 'r')
fout = open('sum.out', 'w')
m = 20000000
arr = []
i = 0
while i<m:
    arr.append(i)
    i = i+1
result = sum(map(int, fin.readline().strip().split()))
fout.write(str(result)+'\n')
